ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -2370,11 +2376,18 @@ snmp_server:
   users:
     - name: < username >
       group: < group_name >
+      # remote_address and udp_port are used for remote users
+      remote_address: < hostname_or_ip_of_remote_engine >
+      # udp_port will not be used if no remote_address is configured
+      udp_port: < udp_port >
       version: < v1 | v2c | v3 >
+      # For a local user (i.e. no remote_ip), use the local engine_id
+      # For a remote user, use the remote engine_id
+      localized: < engine_name in hex >
```

Comment:
localized: < engine_id in hex >

Reply:
hehe - one of the renaming did not make it through!